src/Task/TwigCsFixer.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Formatter\ProcessFormatterInterface;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Config\ConfigOptionsResolver;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use GrumPHP\Fixer\Provider\FixableProcessResultProvider;
+use Symfony\Component\Process\Process;
+
+/**
+ * @extends AbstractExternalTask<ProcessFormatterInterface>
+ */
+class TwigCsFixer extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): ConfigOptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'path' => '.',
+            'triggered_by' => ['twig'],
+        ]);
+
+        $resolver->addAllowedTypes('path', ['string']);
+
+        return ConfigOptionsResolver::fromOptionsResolver($resolver);
+    }
+
+    public function canRunInContext(ContextInterface $context): bool
+    {
+        return $context instanceof GitPreCommitContext || $context instanceof RunContext;
+    }
+
+    public function run(ContextInterface $context): TaskResultInterface
+    {
+        $config = $this->getConfig()->getOptions();
+
+        $files = $context->getFiles()->extensions($config['triggered_by']);
+        if (0 === \count($files)) {
+            return TaskResult::createSkipped($this, $context);
+        }
+
+        $arguments = $this->processBuilder->createArgumentsForCommand('twig-cs-fixer');
+        $arguments->add('lint');
+        $arguments->add('--report=text');
+
+        if ($context instanceof GitPreCommitContext) {
+            $arguments->addFiles($files);
```

Comment:
This could be lots of files, causing CLI max line limits.
Is this the only way to pass files or is there a different way (through sdin, tmp file, ...) ?

Reply:
I've been looking for other ways, but I think this is the only one. I have little experience with this kind of stuff, so maybe I'm missing something...